docs/DeveloperGuide.md

```diff
@@ -765,23 +765,94 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Creating a volunteer
+
+1. Prerequisites: NIL.
+
+    1. Test case: `vcreate n/Little Johnny p/98765432 e/ben@example.com s/chef`<br>
+       Expected: Volunteer is created and added to the bottom of the volunteer list. Details of the volunteer shown in the status message.
+
+    1. Test case: `vcreate n/Ben p/98765432 e/ben@example.com s/chef`<br>
```

Comment:
test case here fails since there is no Ben created yet

Reply:
hmm so in my update-JSON-files PR I just updated the EventStorage and VolunteerStorage json files.

So users who open our app for the first time should see a Ben in the volunteer list

Reply:
does that make sense? or should I explicity create another Ben beforehand

Reply:
oki nvm I changed it to run the vcreate command twice using the same name

Reply:
lmao works